linode_api4/objects/vpc.py

```diff
@@ -1,7 +1,37 @@
+from dataclasses import dataclass
 from typing import Optional
 
 from linode_api4.errors import UnexpectedResponseError
 from linode_api4.objects import Base, DerivedBase, Property, Region
+from linode_api4.objects.serializable import JSONObject
+
+
+@dataclass
+class VPCSubnetLinodeInterface(JSONObject):
+    id: int = 0
+    active: bool = False
+
+
+@dataclass
+class VPCSubnetLinode(JSONObject):
+    id: int = 0
+    interfaces: [VPCSubnetLinodeInterface] = None
+
+    def _serialize(self):
+        """
+        Overrides _serialize to transform interfaces into json
+        """
+        partial = DerivedBase._serialize(self)
+        interfaces = []
+
+        for c in self.interfaces:
+            if isinstance(c, VPCSubnetLinodeInterface):
+                interfaces.append(c.dict)
+            else:
+                interfaces.append(c)
+
+        partial["interfaces"] = interfaces
+        return partial
```

Comment:
This method is not needed as we used [`asdict`](https://docs.python.org/3/library/dataclasses.html#dataclasses.asdict) in the base class.
```suggestion
```